src/visualCaptcha/Captcha.php

```diff
@@ -187,8 +187,8 @@ public function validateImage( $sentOption ) {
     // Validate the sent audio value with the validAudioOption
     public function validateAudio( $sentOption ) {
         $validAudioOption = $this->getValidAudioOption();
-
-        return ( $sentOption == $validAudioOption[ 'value' ] );
+        $validAnwser = explode("|", strtoupper($validAudioOption[ 'value' ]));
```

Comment:
Can you please rename `$validAnwser` to `$validAnswers` so it's clearer it's an array when used below? Also fixing the typo in its name.